sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/FileScan.scala

```diff
@@ -94,22 +96,27 @@ trait FileScan extends Scan with Batch with SupportsReportStatistics with Loggin
   override def hashCode(): Int = getClass.hashCode()
 
   override def description(): String = {
+    val maxMetadataValueLength = 100
```

Comment:
Shall we move this out of functions (`description`/`getMetaData`) because this is used already twice and is assumed that to be the same value, `100`?

Reply:
@dongjoon-hyun Sure. will make the change.